pkg/updater/resultstore/resultstore.go

```diff
@@ -96,8 +210,23 @@ func ResultStoreColumnReader(client *DownloadClient, reprocess time.Duration) up
 			return processedResults[i].InvocationProto.GetTiming().GetStartTime().GetSeconds() > processedResults[j].InvocationProto.GetTiming().GetStartTime().GetSeconds()
 		})
 
+		groups := make(map[string]*invocationGroup, 1)
+
+		updateGroup := func(inv processedResult) *invocationGroup {
```

Comment:
i would prefer to use regular functions over closures with comments - look at `processRawResults` and other functions

Reply:
okay